BackEnd/Backend/Backend.Common/Models/Positions/EditStopLimitRequest.cs

```diff
@@ -0,0 +1,10 @@
+namespace Backend.Common.Models.Positions;
+
+public class EditStopLimitRequest
```

Comment:
This class is unused? i think you should add the StopLimitPrice to EnterPositionRequest and then save it to db